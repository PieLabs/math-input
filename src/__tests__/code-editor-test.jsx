import CodeEditor from '../code-editor';
import React from 'react';
import renderer from 'react-test-renderer';

test('CodeEditor', () => {
  const notReadOnly = renderer.create(<CodeEditor />);
  let tree = notReadOnly.toJSON();
  expect(tree).toMatchSnapshot();
});
