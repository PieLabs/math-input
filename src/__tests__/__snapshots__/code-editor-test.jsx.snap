// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CodeEditor 1`] = `
<textarea
  className="CodeEditor-root-1"
  onBlur={undefined}
  onChange={[Function]}
  tabIndex="-1"
  value={undefined}
/>
`;
